### PR TITLE
Upgrade to richie 2.11.0

### PR DESCRIPTION
**Pull request by sampaccoud**
#### Changed

- Upgrade richie to 2.11.0

####  Fixed

- Upgrade stylesheets to handle new modal styles.

#### Changed

- Upgrade richie to 2.11.0

#### Fixed

- Upgrade stylesheets to handle new modal styles.

#### Changed

- Upgrade richie to 2.11.0

#### Fixed

- Upgrade stylesheets to handle new modal styles.

#### Changed

- Upgrade richie to 2.11.0
- Link footer logo to our new institutional site

#### Fixed

- Upgrade stylesheets to handle new modal styles.

#### Changed

- Upgrade richie to 2.11.0

#### Fixed

- Upgrade stylesheets to handle new modal styles.

#### Changed

- Upgrade richie to 2.11.0

#### Fixed

- Upgrade stylesheets to handle new modal styles.

### CODE REVIEW

File: sites/funmooc/src/backend/templates/richie/base.html [hunk 1]
```diff
@@ -21,4 +21,11 @@
...
-{% block body_js %}{% endblock body_js %}
\ No newline at end of file
+{% block body_js %}{% endblock body_js %}
+
+{% block body_footer_brand %}
+{# Change the link to point to our institutional site #}
+<a href="{% trans "https://www.france-universite-numerique.fr/en/" %}">
+    <img src="{% static "richie/images/logo-alt.png" %}" alt="">
+</a>
+{% endblock body_footer_brand %}
```
Comment (jbpenrath) [inline sites/funmooc/src/backend/templates/richie/base.html:31]: Should we also apply this on FUN Campus and FUN Corporate?

**Review by jbpenrath — COMMENTED** [review]
You could remove `!default` statement within color variable files as these values do not aims to be overridden.

Furthermore I wonder if we should override `body_footer_brand` on fun campus and fun corporate?

File: sites/funmooc/src/backend/templates/richie/base.html (hunk 1 above)
Comment (sampaccoud) [reply sites/funmooc/src/backend/templates/richie/base.html:31]: Yes good idea!

File: sites/ademe/src/frontend/scss/colors/_schemes.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
+// Basic color schemes
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply sites/ademe/src/frontend/scss/colors/_schemes.scss:1]: done

File: sites/ademe/src/frontend/scss/colors/_palette.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    'warm-grey': #747474,
+    'warm-grey-two': #979797,
+    'light-teal': #a8e8e8,
+    'burnt-orange': #bc5500,
+    'pale-grey-two': #eceff1,
+    'pale-grey': #f2f4f5,
+    'pumpkin-orange': #ff7400,
+    'silver': #d5dbe0,
+    'white-three': #fdfdfd,
+  ) !default;
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply sites/ademe/src/frontend/scss/colors/_palette.scss]: done

File: sites/ademe/src/frontend/scss/colors/_gradients.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  r-color('white') 14rem,
+  r-color('white')
+) !default;
+$middle-gradient: linear-gradient(
+  272deg,
+  r-color('dark-aquamarine') 0%,
+  r-color('dark-aquamarine') 99%
+) !default; // seems unused, so fake it
+$dark-gradient: linear-gradient(273deg, r-color('topaz') 1%, r-color('dark-aquamarine') 99%) !default;
+$tealish-gradient: linear-gradient(to left, r-color('tealish'), r-color('light-teal')) !default;
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply sites/ademe/src/frontend/scss/colors/_gradients.scss]: done

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

**Pull request merged**